foodalert/test/test_allergen.py

```diff
@@ -0,0 +1,172 @@
+from django.test import TestCase, Client
+from rest_framework.test import APIRequestFactory, force_authenticate
+#from parameterized import parameterized, param
+from django.contrib.auth.models import User
+from django.db import connection, transaction
+
+import foodalert
+from foodalert.models import Allergen
+from foodalert.serializers import AllergenSerializer
+from foodalert.views import AllergensList
+
+class AllergenTest(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """
+        Sets up a single mock user that can be used for all tests on
+        notification tests
+        """
+        user = "testuser"
+        passw = "test"
+        cls.user = User.objects.create_user(username=user,
+                                            email="testuser@test.com",
+                                            password=passw,
+                                            is_active=1)
+        cls.realAllergen = Allergen.objects.create(
+                name= "test allergen"
+                )
+
+    def setUp(self):
+        self.client.force_login(self.user)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.user.delete()
+        cls.realAllergen.delete()
+
+    def tearDownn(self):
+        pass
+
+    def test_get_allergen_list(self):
+        """
+        This tests that you can read all allergens
+        in the db from a get request. Should return a 
+        200 response upon success
+        """
+        response = self.client.get('/allergen/')
+        self.assertEqual(200, response.status_code)
+        data = response.data[0]
```

Comment:
I think testing against response.content would be a little better here so you can utilize something like 

(Pdb) resp_list = json.loads(response.content)